data/Makefile.am

```diff
@@ -39,8 +39,34 @@ install-sample-data:
 	rm fedora-20-icons.tar.gz; \
 	cd -
 
+com.endlessm.SystemExternalApps.rules: com.endlessm.SystemExternalApps.rules.in
+	$(AM_V_GEN) $(SED)					\
+	-e "s|\@privileged_group\@|$(PRIVILEGED_GROUP)|"	\
+	-e "s|\@helper_user\@|$(EXT_APPS_HELPER_USER)|"		\
+	$< > $@
+
+polkit_rulesdir = $(datadir)/polkit-1/rules.d
+polkit_rules_DATA =					\
+	com.endlessm.SystemExternalApps.rules
+
+%.policy: %.policy.in
+	$(AM_V_GEN) $(MSGFMT) --xml -d $(top_srcdir)/po --template $< -o $@ || cp $< $@
+	$(AM_V_GEN) $(SED)				\
+	-e "s|\@libexecdir\@|$(libexecdir)|"		\
+	$< > $@
+
+polkit_policydir = $(datadir)/polkit-1/actions
+polkit_policy_DATA =					\
+	com.endlessm.SystemExternalApps.policy
+
 EXTRA_DIST =						\
-	$(gsettings_SCHEMAS)
+	$(gsettings_SCHEMAS)				\
+	com.endlessm.SystemExternalApps.policy.in	\
+	com.endlessm.SystemExternalApps.rules.in
+
+DISTCLEAN =						\
```

Comment:
DISTCLEAN is not anything in automake. I think you want `CLEANFILES` since there's no reason they can't be regenerated after a `make clean`.